src/Window.vala

```diff
@@ -266,7 +266,12 @@ namespace Agenda {
         }
 
         public bool privacy_mode_off () {
-            return privacy_setting.get_boolean ("remember-app-usage") || privacy_setting.get_boolean ("remember-recent-files");
+            bool remember_app_usage = privacy_setting.get_boolean
+                ("remember-app-usage");
+            bool remember_recent_files = privacy_setting.get_boolean
+                ("remember-recent-files");
```

Comment:
```diff
-            bool remember_app_usage = privacy_setting.get_boolean
-                ("remember-app-usage");
-            bool remember_recent_files = privacy_setting.get_boolean
-                ("remember-recent-files");
+            bool remember_app_usage = privacy_setting.get_boolean ("remember-app-usage");
+            bool remember_recent_files = privacy_setting.get_boolean ("remember-recent-files");
```

The number of characters in these lines seems to be less than 120 characters, which vala-lint recognizes as too long. So, I don't think we have to have new lines here.